src/components/Mailbox.vue

```diff
@@ -349,6 +364,23 @@ export default {
 			})
 		},
 	},
+	async loadMailbox() {
+		//when the account is unified or inbox, return nothing, else sync the mailbox
+		if (this.account.isUnified || this.folder.specialRole === 'inbox') return
+
+		try {
+			await this.$store.dispatch('syncInboxes')
+
+			logger.debug("Mailbox sync'ed in background")
+		} catch (error) {
+			logger.error('Background sync failed: ' + error.message, {error})
+		}
+		clearInterval(this.loadMailbox)
```

Comment:
this means `loadMailbox` will only be called once, right?

if you use setInterval then only clear it when the component is destroyed.